azure-openai-model-provider/tool.gpt

```diff
@@ -1,6 +1,6 @@
 Name: Azure OpenAI
 Description: Model provider for Azure OpenAI hosted models
-Metadata: envVars: OTTO8_AZURE_OPENAI_MODEL_PROVIDER_API_KEY,OTTO8_AZURE_OPENAI_MODEL_PROVIDER_ENDPOINT,OTTO8_AZURE_OPENAI_MODEL_PROVIDER_DEPLOYMENT_NAME
```

Comment:
Here as well, you are getting rid of it. Pretty sure it's required for chat completions stuff.. is it not working for embeddings or something?

Reply:
The deployment name isn't required. Deployment name is Azure's replacement for model, which is still being passed. My understanding is that only one of deployment name or models needs to be set.

Rest assured that I did test these changes with both chat completions and embeddings and they worked fine.